Bug Fix : Remove -n 4 from ci/testrmsenv.sh

After implementation of issue #91 and running the workflow action, we got the following error: 
```

ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: -n 4
  inifile: /var/local/github/actions-runner02/_work/_temp/fmuconfig-test-root/pyproject.toml
  rootdir: /var/local/github/actions-runner02/_work/_temp/fmuconfig-test-root

Error: Process completed with exit code 4.
```

In order to fix this error we need to remove this line of code of ci/testrmsenv.sh 

```
run_pytest () {
    echo "Running fmu-config tests with pytest..."
    pushd $CI_TEST_ROOT
    pytest ./tests -n 4 -vv 
    popd
}
```

--- a/ci/testrmsenv.sh
+++ b/ci/testrmsenv.sh
@@ -34,6 +34,6 @@ install_test_dependencies () {
 run_pytest () {
     echo "Running fmu-config tests with pytest..."
     pushd $CI_TEST_ROOT
-    pytest ./tests -n 4 -vv 
+    pytest ./tests -vv 
     popd
 }

